frontend/public/components/build-config.tsx

```diff
@@ -213,13 +214,20 @@ export const BuildConfigsPage: React.FC<BuildConfigsPageProps> = (props) => {
       items: allStrategies,
     },
   ];
+
+  const namespace = props.match.params.ns;
```

Comment:
Looks like props is undefined here. The search page is crashing when I select the `BuildConfig` resource from the dropdown.
![Kapture 2022-07-28 at 19 44 38](https://user-images.githubusercontent.com/2561818/181531302-9d8f9cca-2b53-4391-9ff9-0b13f2dc9144.gif)